src/ripple/protocol/impl/Feature.cpp

```diff
@@ -452,6 +452,7 @@ REGISTER_FEATURE(XRPFees,                       Supported::yes, VoteBehavior::De
 REGISTER_FIX    (fixUniversalNumber,            Supported::yes, VoteBehavior::DefaultNo);
 REGISTER_FIX    (fixNonFungibleTokensV1_2,      Supported::yes, VoteBehavior::DefaultNo);
 REGISTER_FIX    (fixNFTokenRemint,              Supported::yes, VoteBehavior::DefaultNo);
+REGISTER_FEATURE(CFTokensV1,                    Supported::yes, VoteBehavior::DefaultNo);
```

Comment:
nit: Should the feature amendment have the full name like`CompactFungibleTokensV1`, similar to how `NonFungibleTokensV1_1` was chosen?

Reply:
i don't think it matters. can change later if necessary but i'm going to leave it for now. note that the feature flags for NFTs mix `NonFungibleToken` and `NFT` anyway